crates/snforge-scarb-plugin/src/attributes/available_gas.rs

```diff
@@ -24,14 +25,32 @@ impl AttributeCollector for AvailableGasCollector {
         args: Arguments,
         _warns: &mut Vec<Diagnostic>,
     ) -> Result<String, Diagnostics> {
-        let &[arg] = args.unnamed_only::<Self>()?.of_length::<1, Self>()?;
-
-        let gas = Number::parse_from_expr::<Self>(db, arg.1, arg.0.to_string().as_str())?;
-
-        let gas = gas.as_cairo_expression();
+        let named_args = args.named_only::<Self>()?;
```

Comment:
Maybe, to facilitate easier migration, we should still allow having a single unnamed attribute? 

The logic would be `either allow single unnamed attribute or only named attributes`. This way it would be backward compatible.

Reply:
Hmm, this is to be consider for sure. 
My rationale for introducing breaking change is that this way _should be encouraged_, and setting just single max fee argument will be removed in next starknet version. Pretty similar situation to returning triplet in gas estimations.

for sure we can discuss :+1:

Reply:
I'm not sure what is the flow here tbh, but I don't think there are attributes that accept multiple types of arguments (named AND unnamed), and I don't that is a good idea to start doing this, so that means we would have to make a new attribute probably. Which might be ok actually. It's just a question if we want this, and I leave the decision up to you :+1:

Reply:
I think at least `fork` supports multiple.

Reply:
Oh shit, you're right. Sorry for my ignorance :( 
Modified the attribute as requested, and added a few tests in newest commit(s)